README.md

```diff
@@ -25,27 +28,24 @@ Only string, numeric, and datetime attributes of Item (`id` and `collection` onl
 may be used to sort results.  
 
 It is not required that implementations support sorting over all attributes, but
-implementations should return a 400 Bad Request status code when attempting to sort over a field name that does
-not support sorting.
-This specification does not yet require the implementation of an "-ables" endpoint (like CQL2 does for queryables)
-that defines the names of the
-fields that can be sorted over, so implementations must provide this out-of-band. Implementers may choose to require
-fields in Item Properties to be prefixed with `properties.` or not, or support use of both the prefixed and non-prefixed
-name, e.g., `properties.datetime` or `datetime`.
-
-Sort behavior may be bound to either or both of
-[STAC API - Item Search](https://github.com/radiantearth/stac-api-spec/tree/v1.0.0/item-search) (`/search` endpoint) or
-[STAC API - Features](https://github.com/radiantearth/stac-api-spec/tree/v1.0.0/ogcapi-features)
-(`/collections/{collectionId}/items` endpoint) by advertising the relevant conformance class.
+implementations should either implement Sortables or just return a 400 Bad Request status code
+when attempting to sort over a field name that does not support sorting.
+Implementers may choose to require fields in Item Properties to be prefixed with `properties.` or not,
+or support use of both the prefixed and non-prefixed name, e.g., `properties.datetime` or `datetime`.
```

Comment:
Higher-level question, is there any way to indicate whether a given client does "implicit prefixes" or not? I've found that, from the "client" perspective, it can be hard to know whether I should prefix with `properties` or not (**pgstac** just added support for both in December: https://github.com/stac-utils/pgstac/pull/329/files#diff-2274efb3ca918a5fdd73a71e90b8bbd97b180963ea131085df5aa0195455c5d6R129).

Reply:
No, not that I'm aware of, but wouldn't you express that in the property names? 

Your properties may be either:

- properties.datetime
- properties.title
- ...

or:

- datetime
- title
- ...

The client should anyway only offer those that are in the list of properties, so it's pretty irrelevant whether they have prefixes or not.

Reply:
> The client should anyway only offer those that are in the list of properties, so it's pretty irrelevant whether they have prefixes or not.

I guess this problem will be at least partially solved by `sortables`, since it will give servcies a way to say "please ask for sorting this way". Before `sortables` it was guess-and-check.

Reply:
Yeah, this is meant to be solved by sortables. For cases where sortables is not available, it's still as it was before. This PR is meant to be additive, not change any existing behaviour for APIs that did implement the previous version.